.travis.yml

```diff
@@ -39,14 +29,22 @@ env:
     - SBT_CMD="mavenResolverPluginTest:scripted dependency-management/*2of2"
     - SBT_CMD="mavenResolverPluginTest:scripted dependency-management/*1of2 project/transitive-plugins"
 
-notifications:
-  email:
-    - eugene.yokota@typesafe.com
-    - joshua.suereth@typesafe.com
+matrix:
+  fast_finish: true
+  include:
+    - env: SBT_CMD="scalafmtDiffTest"
```

Comment:
See https://olafurpg.github.io/scalafmt/#Coursier for more information on the script. The current approach has the disadvantage that we pay the sbt startup time to format, while the CLI tool uses nailgun to reduce startup time of the JVM.